{"name": "test_expect_triangle_to_be_invalid", "status": "passed", "statusDetails": {"message": "XPASS Esperamos que o teste falhe"}, "start": 1693672024302, "stop": 1693672024302, "uuid": "8a676337-d324-4fc4-9d67-1b23c6971928", "historyId": "f6ff257340833d57f855a94b11f5413b", "testCaseId": "f6ff257340833d57f855a94b11f5413b", "fullName": "services.test.test_triangle_cal#test_expect_triangle_to_be_invalid", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(condition=<function <lambda> at 0x0000018429922FC0>, reason='Esperamos que o teste falhe')"}, {"name": "parentSuite", "value": "services.test"}, {"name": "suite", "value": "test_triangle_cal"}, {"name": "host", "value": "Leovski"}, {"name": "thread", "value": "15852-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "services.test.test_triangle_cal"}]}